modules/objdetect/samples/barcode.cpp

```diff
@@ -0,0 +1,223 @@
+#include <iostream>
+#include "opencv2/objdetect/barcode.hpp"
```

Comment:
> `#include "opencv2/objdetect/barcode.hpp"`

Fix and use `#include "opencv2/objdetect.hpp"` instead.

Reply:
Done!